tools/topology/topology1/CMakeLists.txt

```diff
@@ -138,6 +138,7 @@ set(TPLGS
 	"sof-imx8-cs42888-mixer\;sof-imx8-cs42888-mixer"
```

Comment:
I don't see in the last patch what "Use edma copy data." means. You can remove that line from the commit message.

Reply:
Add this just for understanding how the pipeline works on 8ulp. If you think we don't need it, I will remove it.

Reply:
The DMA engine isn't that relevant when describing a pipeline, shouldn't be part of the specification of the pipeline itself. I'd say remove that line.

Reply:
Removed.